modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialog.ts

```diff
@@ -264,64 +266,145 @@ module app.create {
         }
 
         private loadContentTypes() {
+
+            this.showLoadingMasks();
+
+            wemQ.all(this.prepareRequestsToFetchContentData())
+                .spread((contentTypes: ContentTypeSummary[], directChilds: api.content.ContentResponse<api.content.ContentSummary>,
+                         parentSite: Site) => {
+
+                    this.listItems = this.createListOfContentTypeItems(contentTypes, parentSite);
+                    this.mostPopularItems = this.findMostPopularContentTypes(2, contentTypes, directChilds.getContents());
```

Comment:
It's better to move itemsToFind number like a default constant to MostPopularItemsList, then easy will to check and change it